packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -389,6 +390,9 @@ class DropdownMenu<T> extends StatefulWidget {
   /// which contains the contents of the text input field.
   final SearchCallback<T>? searchCallback;
 
+  /// {@macro flutter.widgets.editableText.inputFormatters}
```

Comment:
Seems using this directly will cause the doc error because this class doesn't contain `onChanged` property. We can just copy the doc here and remove that part.

Reply:
thank you for finding this!